.github/workflows/workflow.yml

```diff
@@ -35,6 +35,10 @@ jobs:
         run: |
           python -m unittest
 
+      - name: Run linter
+        run: |
+          flake8 --config=setup.cfg u8timeseries
```

Comment:
does this automatically apply the changes on the code?

Reply:
@radujica is it failing on some warnings or errors or correct them? I believe former but double-checking :)

Reply:
Can we run it before tests or even as separate step so we can run it simultaneously? Unless it is < 2 secs then ignore it

Reply:
It's like 1s so easiest to just keep it here.

It doesn't correct anything by itself, just fails the github action if an issue is found (after printing what failed, ofc).

IMO we shouldn't let some external tool edit our code and commit it, e.g. through black lib. It also subtly pushes for a proper dev environment if developing many things ;)